build.py

```diff
@@ -70,6 +74,8 @@ def main():
                         name if is_leaf else label,
                     )
                 )
+                if i == 0 and is_leaf:
```

Comment:
We also need to check that `include_subdomains` is truthy here as well.

Reply:
I forgot this check. Fixed in 2c7abdc